Measurements/Tests.qs

```diff
@@ -22,9 +22,10 @@ namespace Quantum.Kata.Measurements {
 
     // "Framework" operation for testing single-qubit tasks for distinguishing states of one qubit
     // with Bool return
-    operation DistinguishTwoStates_OneQubit (statePrep : ((Qubit, Int) => Unit), testImpl : (Qubit => Bool)) : Unit {
+    operation DistinguishTwoStates_OneQubit (statePrep : ((Qubit, Int) => Unit), testImpl : (Qubit => Bool), str1 : String, str_aim : String) : Unit {
         let nTotal = 100;
-        mutable nOk = 0;
+        mutable nNOTOk0as1 = 0;
```

Comment:
Similarly here, I'd make this an array `misclassifications` and store how many of states with the given index were misclassified. Then lines 43-48 become just `set misclassifications[state] += 1;`

Reply:
Thanks for the tip. Could not find a way to += in Q#, instead I will use:
                    set misclassifications w/= state <-  misclassifications[state]+1;

Reply:
True, I keep forgetting the array element update syntax...